src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-# Changelog
+# Changelog - OpenTelemetry.Exporter.Geneva
```

Comment:
You don't need to add the project name here as this CHANGELOG file already exists under the directory `OpenTelemetry.Exporter.Geneva`

Look at other projects in the solution for reference to stay consistent.

Reply:
Looks like others are doing this https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Contrib.Extensions.AWSXRay/CHANGELOG.md#changelog---opentelemetrycontribextensionsawsxray

Reply:
And it is very inconsistent just by quickly looking at 3 projects.
I do agree that it's better to just keep it as "Changelog".

Reply:
Ahh I wasn't aware of this earlier.

Reply:
Ya was just going off another project but if that's not something followed everywhere I agree to omit the project name :)